src/tutorials/chat_commands_tutorial.md

```diff
@@ -0,0 +1,2146 @@
+        else
+            status.senderMessage = CommandParser.error.NO_PERMISSION
+        end
+    end)
+
+    -- /endround
+    CommandParser.AddCommand("endround", function(sender, params, status)
+        if CommandParser.HasPermission(sender, CommandParser.permissions.ADMIN) then
+            if Game.GetTeamScore(1) > Game.GetTeamScore(2) then
+                Events.Broadcast("TeamVictory", 1)
+            elseif Game.GetTeamScore(2) > Game.GetTeamScore(1) then
+                Events.Broadcast("TeamVictory", 2)
+            else
+                Events.Broadcast("TieVictory")
+            end
+
+            ABGS.SetGameState(ABGS.GAME_STATE_ROUND_END)
+        else
+            status.senderMessage = CommandParser.error.NO_PERMISSION
+        end
+    end)
+    ```
+
+## Fun Commands
+
+In this section you will create some fun commands that you can use on yourself or other players for a bit of fun.
+
+### Push
+
+This command will allow you to push players around. If you see them on the edge of a platform, give them a friendly nudge.
+
+By getting setting the player's velocity combined with a direction, it can be easy to launch a player in a certain direction.
+
+```lua
+-- /push player direction amount
+CommandParser.AddCommand("push", function(sender, params, status)
+    if CommandParser.HasPermission(sender, CommandParser.permissions.ADMIN) then
+        local who = CommandParser.GetPlayer(CommandParser.ParamIsValid(params[2]))
+
+        if who ~= nil then
+            local direction = CommandParser.ParamIsValid(params[3])
+            local currentVel = who:GetVelocity()
+            local dirVector = who:GetWorldTransform():GetUpVector()
+            local amount = tonumber(params[3]) or 2000
+
+            if direction == "left" then
+                dirVector = -who:GetWorldTransform():GetRightVector()
+            elseif direction == "right" then
+                dirVector = who:GetWorldTransform():GetRightVector()
+            elseif direction == "forward" then
+                dirVector = who:GetWorldTransform():GetForwardVector()
+            elseif direction == "back" then
+                dirVector = -who:GetWorldTransform():GetForwardVector()
+            end
+
+            who:SetVelocity(currentVel + dirVector * amount)
+        else
+            status.senderMessage = CommandParser.error.INVALID_PLAYER
+        end
+    else
+        status.senderMessage = CommandParser.error.NO_PERMISSION
+    end
+end)
+```
+
+<div class="mt-video" style="width:100%">
+    <video autoplay muted playsinline controls loop class="center" style="width:100%">
+        <source src="/img/ChatCommands/fun_push_command.mp4" type="video/mp4" />
+    </video>
+</div>
+
+### Ragdoll
+
+Ragdoll physics can be a lot of fun. This command will enable the ragdoll physics for a few sockets.
+
+By calling the `EnableRadoll` function on the player, you can specify a socket and weight.
+
```

Comment:
perhaps mention the animations and sockets reference here

side note: would be funny to have a dance command